project-handbook/technical-infrastructure.md

```diff
@@ -0,0 +1,63 @@
+# Overview of OpenRail's Technical Infrastructure
+
+The technical infrastructure of the OpenRail Association is comprised of various components to enable the work of the OpenRail projects as well as of different working groups such as the *OpenRail Team*.
+
+For the composition, we follow these principles:
+
+* We prefer Open Source
+* We chose software and services based on whether they are feasible and pragmatic
+* We prefer using hosted services instead of self-hosting
+* We strive for ease of access and use for the respective users and teams
+* We appreciate modularity, and accept incremental improvements when perfection is not possible
+
+## Governance
+
+The *Technical Infrastructure Manager*, part of the OpenRail Team, is responsible for the overall infrastructure. They do this in close collaboration with the *Technical Committee* as well as other users inside the organisation. They can define assistants to avoid that all knowledge and accesses reside with only one person.
+
+The current Technical Infrastructure Manager is Max Mehl ([@mxmehl](https://github.com/@mxmehl)). He is assisted by Cornelius Schumacher ([@cornelius](https://github.com/@cornelius)).
+
+## Domains
+
+OpenRail Association registers and manages all of its domains via [INWX](https://www.inwx.com/), a Germany-based domain registrar. This also contains domains by OpenRail projects.
+
+INWX supports sub-accounts which may be used to allow tools and other persons to access only specific parts of the account.
+
+Projects may control their domains on their own. Please see [this document](./domain-names.md) for guidance and details.
+
+## GitHub
+
+OpenRail Association prefers using GitHub.com for hosting of code, projects and documentation. All official repositories - e.g., by OpenRail projects, working groups, for administrative projects - must be part of the [OpenRailAssociation](https://github.com/OpenRailAssociation) GitHub organisation.
```

Comment:
nit: I believe GitHub uses the "organization" spelling (international?), we might want to keep a consistent style

Reply:
I sticked to british english for this document. AFAIK we didn't talk about whether to use AE or BE inside OpenRail Association.

For this part, I consider it fine to use "organisation" as it's just the BE way of spelling this, and I don't believe GitHub's "organization" is a term in itself.